assets/scripts/app/keypress.js

```diff
@@ -213,7 +162,7 @@ export function registerKeypress (commands, options, callback) {
       command.originalCommands = originalCommands
 
       // Special case for 'ESC' key; it defaults to global (window) focus
-      if (Number.parseInt(keyCode, 10) === KEYS['esc']) {
+      if (key === 'Escape') {
```

Comment:
This will need the IE-specific `Esc` value as well.

Reply:
Added check for `Esc`